asyncstdlib/_core.py

```diff
@@ -93,25 +93,25 @@ async def borrow(iterator: AsyncIterator[T]) -> AsyncGenerator[T, None]:
 
 
 def awaitify(
-    function: Union[Callable[..., T], Callable[..., Awaitable[T]]]
-) -> Callable[..., Awaitable[T]]:
+    function: Union[Callable[P, T], Callable[P, Awaitable[T]]]
+) -> Callable[P, Awaitable[T]]:
```

Comment:
For types that are not valid at runtime on all versions, quote the type:
```suggestion
    function: "Union[Callable[P, T], Callable[P, Awaitable[T]]]"
) -> "Callable[P, Awaitable[T]]":
```